pkg/validations/createvalidations/gitops.go

```diff
@@ -10,9 +10,9 @@ import (
 	"github.com/aws/eks-anywhere/pkg/validations"
 )
 
-func ValidateGitOpsNameIsUnique(ctx context.Context, k validations.KubectlClient, cluster *types.Cluster, spec *cluster.Spec) error {
+func ValidateGitOps(ctx context.Context, k validations.KubectlClient, cluster *types.Cluster, spec *cluster.Spec) error {
```

Comment:
hm, `ValidateGitOps` is a pretty broad name, and this method only validates the relationship between a Mgmt cluster and workload cluster gitops file structure. `ValidateGitOpsDirStructure`? `ValidateGitOpsRootPath`? 

Or keep `ValidateGitOps` as the top-level method and make a sub-method like `validateGitOpsRootPath` that's called here, so it's clear what we're validating

Reply:
updated!